### PR TITLE
github-action/riscv: enable gtest

### PR DESCRIPTION
**Pull request by myungjoo**
Enable unittesting with gtest

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

**Review by songgot — APPROVED** [review]
LGTM

File: .github/workflows/risc-v.yml [hunk 1]
```diff
@@ -19,6 +19,15 @@ jobs:
           githubToken: ${{ github.token }}
           run: |
             apt-get -qy update
-            apt-get -qy install meson ninja-build libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libglib2.0-dev libjson-glib-dev gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-good libgtest-dev libpng-dev libopencv-dev python3 python3-dev python3-numpy pkg-config gcc g++ liblua5.1-dev
+            apt-get -qy install meson ninja-build libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libglib2.0-dev libjson-glib-dev gstreamer1.0-tools gstreamer1.0-plugins-base gstreamer1.0-plugins-good libgtest-dev libpng-dev libopencv-dev python3 python3-dev python3-numpy pkg-config gcc g++ liblua5.1-dev libgtest-dev bash
```
Comment (jaeyun-jung) [inline .github/workflows/risc-v.yml:22]: pkg libgtest-dev is duplicated.

File: .github/workflows/risc-v.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@ on:
 jobs:
   build_job:
     runs-on: ubuntu-20.04
-    name: Build on alpine RISC-V
+    name: Build on Ubuntu 18.04 RISC-V 64
```
Comment (jaeyun-jung) [inline .github/workflows/risc-v.yml:10]: on 20.04?

Comment (myungjoo) [reply .github/workflows/risc-v.yml]: fixed.

File: .github/workflows/risc-v.yml (hunk 1 above)
Comment (myungjoo) [reply .github/workflows/risc-v.yml]: fixed.

**Pull request merged**